cfme/tests/configure/test_access_control.py

```diff
@@ -2577,7 +2577,38 @@ def test_authorized_users_can_login():
             1. User created
             2. User logged in
     """
-    pass
+    # creating role
```

Comment:
I believe we already have automated coverage for this test.

https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/integration/test_db_auth.py#L26

test_db_auth is creating a user with a credential randomized by fauxfactory.gen_alpha, which will include lower and uppercase characters.

It doesn't _always_ include an uppercase character, as its randomized.

I think the existing test cases and fixture should be modified to always include a capital letter in the credential, than create a new test case for this.